packages/page-claims/src/index.tsx

```diff
@@ -80,7 +79,7 @@ const ClaimsApp = (props: Props): React.ReactElement => {
   const [isPreclaimed, setIsPreclaimed] = useState(false);
 
   useEffect(() => {
-    if (isApiReady && typeof api.query.claims.claimAttest !== undefined) {
+    if (isApiReady && typeof api.query.claims.claimAttest !== 'undefined') {
```

Comment:
Is there a guarantee that isApiReady will be `true` before rendering this component? Like somewhere higher in the components tree?

If yes, we could simplify this useEffect&useState into just
```ts
const isOldClaimProcess = !!api.query.claims.claimAttest;
```